tests/io/v3/assets/test_assets_api.py

```diff
@@ -0,0 +1,460 @@
+def test_asset_import(api):
+    source = 'example_source'
+    assets = {
+        'fqdn': ['example.py.test'],
+        'ipv4': ['192.168.254.1'],
+        'netbios_name': 'example',
+        'mac_address': ['00:00:00:00:00:00'],
+    }
+    resp_data = {'asset_import_job_id': '467e5338-7783-4a0d-915a-5d00584784a0'}
+    responses.add(
+        responses.POST, url=f'{ASSET_BASE_URL}/import', json=resp_data
+    )
+
+    data = api.v3.assets.asset_import(source, assets)
+    assert data == resp_data['asset_import_job_id']
+
+
+@responses.activate
+def test_list_import_jobs(api):
+    resp_data = {
+        'asset_import_jobs': [
+            {
+                'job_id': 'b9584671-68e6-426b-a67c-6373778b8a0a',
+                'container_id': 'cfdabb09-6aef-481d-b28f-aecb1c38f297',
+                'source': 'my_source',
+                'batches': 1,
+                'uploaded_assets': 1,
+                'failed_assets': 0,
+                'start_time': '2021-11-24T13:43:56.709Z',
+                'last_update_time': '2021-11-24T13:43:56.709Z',
+                'end_time': '2021-11-24T13:43:56.709Z',
+                'status': 'COMPLETE',
+                'status_message': '',
+            },
+            {
+                'job_id': 'fb57df61-cc1c-4c2c-9a76-1f0148c1949a',
+                'container_id': 'cfdabb09-6aef-481d-b28f-aecb1c38f297',
+                'source': 'my_source',
+                'batches': 1,
+                'uploaded_assets': 1,
+                'failed_assets': 0,
+                'start_time': '2021-11-24T13:43:56.709Z',
+                'last_update_time': '2021-11-24T13:43:56.709Z',
+                'end_time': '2021-11-24T13:43:56.709Z',
+                'status': 'COMPLETE',
+                'status_message': '',
+            },
+        ]
+    }
+    responses.add(
+        responses.GET, url=f'{ASSET_BASE_URL}/import/jobs', json=resp_data
+    )
+
+    data = api.v3.assets.list_import_jobs()
+    assert data == resp_data['asset_import_jobs']
+
+
+@responses.activate
+def test_import_job_details(api):
+    job_id = 'b9584671-68e6-426b-a67c-6373778b8a0a'
+    resp_data = {
+        'job_id': 'b9584671-68e6-426b-a67c-6373778b8a0a',
+        'container_id': 'cfdabb09-6aef-481d-b28f-aecb1c38f297',
+        'source': 'my_source',
+        'batches': 1,
+        'uploaded_assets': 1,
+        'failed_assets': 0,
+        'start_time': '2021-11-24T13:43:56.709Z',
+        'last_update_time': '2021-11-24T13:43:56.709Z',
+        'end_time': '2021-11-24T13:43:56.709Z',
+        'status': 'COMPLETE',
+        'status_message': '',
+    }
+    responses.add(
+        responses.GET,
+        url=f'{ASSET_BASE_URL}/import/jobs/{job_id}',
+        json=resp_data,
+    )
+
+    data = api.v3.assets.import_job_details(job_id)
+    assert data == resp_data
+
+
+@responses.activate
+def test_move_assets(api):
+    source = 'b9584671-68e6-426b-a67c-6373778b8a0a'
+    destination = 'b7584671-68e6-426b-a67c-6373778b8a0a'
```

Comment:
set destination to different network-uuid to avoid confusion.

Reply:
these are different if u look closely. source starts with 'b9' where destination with 'b7'